docs/src/modules/components/Notifications.js

```diff
@@ -53,34 +59,34 @@ export default function Notifications() {
   const messages = useSelector((state) => state.notifications.messages);
   const lastSeen = useSelector((state) => state.notifications.lastSeen);
 
-  const messageList = (messages || [])
-    .filter((message) => {
-      if (
-        message.userLanguage &&
-        message.userLanguage !== userLanguage &&
-        message.userLanguage !== navigator.language.substring(0, 2)
-      ) {
-        return false;
-      }
-      return true;
-    })
-    .reverse();
-
-  const unseenNotificationsCount = messageList.reduce(
-    (count, message) => (message.id > lastSeen ? count + 1 : count),
-    0,
-  );
+  const messageList = messages
```

Comment:
```suggestion
  const messageList = Array.isArray(messages)
```

?

Reply:
What's the advantage?

Reply:
It’s more explicit but ultimately doesn’t need to happen.